### PR TITLE
Add a project for WeTek Core TV-box

### PR DESCRIPTION
**Pull request by codesnake**

### CODE REVIEW

**Comment by sraue** [comment]
great, can you update the license headers? it should not show the postal adress, instead the emailadress from FSF and the dates should be updated

**Comment by codesnake** [comment]
Done

**Comment by sraue** [comment]
thanks much :-)

File: projects/WeTek_Core/patches/kodi/0003-aml-Add-support-for-4k-resolutions.patch [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
++void CEGLNativeTypeAmlogic::GetMaxResolution(RESOLUTION_INFO &maxResolution)
++{
++  std::vector<RESOLUTION_INFO> resolutions;
++  ProbeResolutions(resolutions);
++
++  maxResolution = {0};
++  for (size_t i = 0; i < resolutions.size(); i++)
++  {
++    if (resolutions[i].iScreenWidth > maxResolution.iScreenWidth || resolutions[i].iScreenHeight > maxResolution.iScreenHeight)
++      maxResolution = resolutions[i];
```
Comment (fritsch) [inline projects/WeTek_Core/patches/kodi/0003-aml-Add-support-for-4k-resolutions.patch]: Be careful here, that would always choose 4096x2160 on 4k display, but native resolution is 3840x2160
Also in general you cannot play 1080i50 liveTV with deinterlacing on a 4k@30p screen

File: projects/WeTek_Core/patches/kodi/0005-aml-Fix-compiler-badness-when-compiling-with-amcodec.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+index 56ec6a3..52fa07e 100644
+--- a/xbmc/cores/dvdplayer/DVDCodecs/Video/Makefile.in
++++ b/xbmc/cores/dvdplayer/DVDCodecs/Video/Makefile.in
+@@ -26,8 +26,6 @@ endif
+ ifeq (@USE_LIBAMCODEC@,1)
+ SRCS += AMLCodec.cpp
+ SRCS += DVDVideoCodecAmlogic.cpp
+-INCLUDES += -I$(prefix)/include/amcodec
+-INCLUDES += -I$(prefix)/include/amplayer
+ endif
```
Comment (fritsch) [inline projects/WeTek_Core/patches/kodi/0005-aml-Fix-compiler-badness-when-compiling-with-amcodec.patch]: Something for upstream?

Comment (codesnake) [inline projects/WeTek_Core/patches/kodi/0005-aml-Fix-compiler-badness-when-compiling-with-amcodec.patch]: Yes, will push shortly.

**Pull request merged**